src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -1081,6 +1092,18 @@ class EventView {
     // we just use the default display mode
     return DisplayModes.DEFAULT;
   }
+
+  getQueryWithOverrides() {
+    const {query} = this;
+    const conditions = tokenizeSearch(query);
```

Comment:
You could move this to 1101 and save a tokenize when there are no additionalConditions ⛳